Marketplace/whiteboard/whiteboardManagerServer.js

```diff
@@ -0,0 +1,508 @@
+            var color = utils.utils.parseJSON(params[2]);
+
+            if ( markerName === "hifi_model_marker_blue" && 
+                (blueMarkerID === null || params[0] === blueMarkerID)) {
+                blueMarkerID = _this.spawnMarkerWithColor(markerName, color);
+            } else if ( markerName === "hifi_model_marker_green" && 
+                (greenMarkerID === null || params[0] === greenMarkerID)) {
+                greenMarkerID = _this.spawnMarkerWithColor(markerName, color);
+            } else if ( markerName === "hifi_model_marker_black" && 
+                (blackMarkerID === null || params[0] === blackMarkerID)) {
+                blackMarkerID = _this.spawnMarkerWithColor(markerName, color);
+            } else if ( markerName === "hifi_model_marker_red" && 
+                (redMarkerID === null || params[0] === redMarkerID)) {
+                redMarkerID = _this.spawnMarkerWithColor(markerName, color);
+            } else if ( markerName === "hifi_model_marker_pink" && 
+                (pinkMarkerID === null || params[0] === pinkMarkerID)) {
+                pinkMarkerID = _this.spawnMarkerWithColor(markerName, color);
+            } else if ( markerName === "hifi_model_marker_yellow" && 
+                (yellowMarkerID === null || params[0] === yellowMarkerID)) {
+                yellowMarkerID = _this.spawnMarkerWithColor(markerName, color);
+            }
+        },
+        /// Remotely callable function that creates a new eraser
+        /// 
+        /// Attempts to spawn a new eraser.
+        /// This function is called when an original eraser is grabbed. 
+        /// 
+        /// @param {string} entityID of the server
+        /// @param {object} params [grabbedEraserID]
+        spawnEraser: function(entityID, params) {
+            if (eraserID === null || params[0] === eraserID) {
+                var template = getTemplate(ERASER_NAME);
+                var rootPos = Entities.getEntityProperties(_this.entityID, "position").position;
+                var currentRot = Entities.getEntityProperties(_this.entityID, "rotation").rotation;
+                var rootRot = getTemplate("Whiteboard")['rotation'];
+                var localPos = template['localPosition'];
+                var up = Vec3.multiply(Quat.getUp(rootRot), Vec3.dot(localPos, Quat.getUp(rootRot)));
+                var front = Vec3.multiply(Quat.getFront(rootRot), Vec3.dot(localPos, Quat.getFront(rootRot)));
+                var right = Vec3.multiply(Quat.getRight(rootRot), Vec3.dot(localPos, Quat.getRight(rootRot)));
+                var relativePosInWorld = Vec3.sum(Vec3.sum(up, right), front);
+                relativePosInWorld = Vec3.multiplyQbyV(currentRot, relativePosInWorld);
+                var finalPosition = Vec3.sum(relativePosInWorld, rootPos);
+                eraserID = Entities.addEntity( {
+                    parentID: "{00000000-0000-0000-0000-000000000000}",
```

Comment:
` parentID: "{00000000-0000-0000-0000-000000000000}",`   can be removed since its the default value of parentID